src/libraries/System.Private.Runtime.InteropServices.JavaScript/src/System/Runtime/InteropServices/JavaScript/EnumExport.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace System.Runtime.InteropServices.JavaScript
+{
+    public enum ConvertEnum
+    {
+        Default,
+        ToLower,
+        ToUpper,
+        Numeric
+    }
+
+    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Property | AttributeTargets.Method | AttributeTargets.Field,
+            AllowMultiple = true, Inherited = false)]
+    public class EnumExportAttribute : Attribute
+    {
+        public EnumExportAttribute() : this(null)
+        {
+        }
+
+        public EnumExportAttribute(string? contractName)
+        {
+            ContractName = contractName;
+        }
+
+        public string? ContractName { get; }
+        public ConvertEnum EnumValue { get; set; }
```

Comment:
I'm still not convinced this extra logic dimension is useful

/cc @lewing

Reply:
This has all been simplified now.  

Simplify the export interface. 
- remove extra logic dimension
- remove unnecessary dependency on reflection and metadata
- default is to now return numeric value if no attribute is defined.